supported_platforms.md

```diff
@@ -0,0 +1,55 @@
+# Microsoft Azure IoT SDKs for Java
+
+This SDK is tested nightly on Windows 10, Ubuntu 1604, and on an Android emulator. For additional details
+for each tested platform, see the respective sections below. 
+
+While this SDK contains several different packages, each package supports the same platforms and JDKs.
+
+## Supported JDK versions
+
+Currently, this SDK supports JDK versions 8 and 11. This SDK may work with other versions, but there are no guarantees 
+that they will always work for those versions nor are there guarantees that we will fix bugs that are only present on
+those versions.
+
+
+## Windows 10
+
+Nightly test platform details:
+Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
+Java version: 1.8.0_292, vendor: AdoptOpenJDK, runtime: C:\hostedtoolcache\windows\Java_Adopt_jdk\8.0.292-10\x64\jre
```

Comment:
I don't think the runtime info lends any extra information since we already have the version no specified. Do you think there is any value in including the runtime info?

Reply:
Nope, I agree.